pom.xml

```diff
@@ -12,7 +12,21 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<!-- tomcat version - version 7 recommended, supports java 1.6+ -->
+		<!-- tomcat jars need to be in the repo though -->
 		<tomcat.version>6.0.26</tomcat.version>
+		<!-- 
+		<tomcat.version>7.0.67</tomcat.version>		
+		 -->
+		 <!-- tomcat version -->
+		<openmrs.version>2.0.0-SNAPSHOT</openmrs.version>
+		<!-- use -D to specify the os_arch to select mysqld -->
+		<!-- 
+		<os.name>Win</os.name>
+		<os.arch>x86</os.arch>
+		 -->
+		<!-- os_arch -->
+		<javaVersion>1.8</javaVersion>
 	</properties>
```

Comment:
the only change necessary is to use javaVersion 1.8 instead of the currently specified 1.6. If this change is made then maybe it maybe necessary to change the jar plugins.
If (and I clearly don't understand this very well) it is not necessary to change the compiler version in the pom, and your pull request works then this pull request can be closed without merging.
I would appreciate if you explain in a comment why we compile with java 1.8 but leave the compiler spec in the pom as java 1.6. Thanks.